mypostcard/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"bytes"
+	"embed"
```

Comment:
I would highlight this import by putting it in its own block and add a comment explaining the importance of this import.